torch/csrc/Module.cpp

```diff
@@ -526,6 +532,33 @@ PyObject *THPModule_benchmarkCuDNN(PyObject *_unused, PyObject *noargs)
   Py_RETURN_FALSE;
 }
 
+PyObject *THPModule_setBenchmarkLimitCuDNN(PyObject *_unused, PyObject *arg)
+{
+  THPUtils_assert(THPUtils_checkLong(arg), "set_benchmark_limit_cudnn expects an int, "
+          "but got %s", THPUtils_typename(arg));
+  auto benchmark_limit = static_cast<int>(THPUtils_unpackLong(arg));
+#if defined(USE_ROCM)
+  if (benchmark_limit != 0 && benchmark_limit != 10) {
```

Comment:
should you warn unconditionally here? This can only be a result of user calling `set_benchmark_limit`, so whatever value is passed, ROCM should warn.